src/lib/components/Toast.svelte

```diff
@@ -7,7 +7,7 @@
 </script>
 
 <div
-	transition:fade={{ duration: 300 }}
+	transition:fade|global={{ duration: 300 }}
```

Comment:
it's the default now?

Reply:
`|local` is the default now and `|global` needs to be specified. This used to be the opposite!